README.md

```diff
@@ -26,35 +26,35 @@
 
 ## Introduction
 
-This is the main repository to maintain the libraries' assets on CDNJS. For website and API, please refer to [new-website](https://github.com/cdnjs/new-website) repository, all the repositories can be found at [CDNJS](https://github.com/cdnjs/) organization on GitHub!
+This is the main repository to maintain the libraries' assets on CDNJS. For our website and API, please refer to the [new-website](https://github.com/cdnjs/new-website) repository. You can find all repositories at [CDNJS](https://github.com/cdnjs/) on GitHub!
 
-CDNJS is a free and open source project to organize all the famous web front-end development resources and provide them to the developers with faster CDN infrastructure without usage limitation and fee. We want to help individual library/framework developers spread their projects, and help web developers to supercharge their websites! With the great and free CDN service, developers can focus on the project and website development, without spending time worrying about how to setup a CDN for the project or website assets. We hope to make web development easier, your websites and the WWW faster and safer.
+CDNJS is a free open-source project to organize and provide all famous front-end web development resources to developers via a fast CDN infrastructure without usage limitations and fees. We want to help individual library/framework developers distribute their projects, and web developers supercharge their websites! With our great free CDN service, developers can focus on their projects and website development. Developers no longer have to spend time worrying about how to set-up a CDN for projects or website assets. We hope to make web development easier, as well as your websites and the WWW faster and safer.
```

Comment:
hmmm, I suppose we usually use the common term `free and open source project` instead of `free open-source project`?

Reply:
Hi @PeterDaveHello. Sure, I can fix that if you like. Also, in my PR I capitalized the acronym url to URL in all cases in the README body except for in the Table of Contents. The comments mentioned not to touch this section manually. Should I keep this change or switch it back to how it was beforehand to be consistent with the Table of Contents item?

Reply:
Thanks, let's change it back!

It's fine if you'd like to update the Table of Contents :)

Reply:
Ok, great. Just one last question from a beginner. Should I squash the new commit with the above changes with my first commit on my branch after I make my changes?

Reply:
@LeaSak Yes, we hope the commits can be as clean as possible. Could you please squash them? Thanks.

Reply:
Thanks @maruilian11 for helping me. I just squashed the commits.